test/evp_test.c

```diff
@@ -1798,6 +1801,7 @@ static int encode_test_run(EVP_TEST *t)
     ENCODE_DATA *expected = t->data;
     unsigned char *encode_out = NULL, *decode_out = NULL;
     int output_len, chunk_len;
+    EVP_ENCODE_CTX *encode_ctx = NULL;
     EVP_ENCODE_CTX *decode_ctx;
```

Comment:
Nit:  I know the NULL initialization is not necessary for `decode_ctx`. But I have an obsessive addiction to consistency: Why not join the two lines as

```C
EVP_ENCODE_CTX *encode_ctx = NULL, *decode_ctx = NULL;
```` 
to make it consistent with L1802?

https://github.com/openssl/openssl/blob/e89d6956eef73333f4ed6b6858e948a8cda40e96/test/evp_test.c#L1802

But if you don't share my obsession, you are free to ignore this comment ;-)